src/index.html

```diff
@@ -7,24 +7,26 @@
     <title>Document</title>
   </head>
   <body>
-    <header><h1>Leaderboard</h1></header>
-    <main>
-      <section>
-        <div class="get">
-          <h2>Recent scores</h2>
-          <button type="reset">Refresh</button>
-        </div>
-        <ol></ol>
-      </section>
-      <section>
-        <div class="post">
-          <h2>Add your score</h2>
-          <input id="name" type="text" placeholder="Your name" />
-          <input id="score" type="text" placeholder="Your score" />
-          <button type="submit">Submit</button>
-        </div>
-      </section>
-    </main>
-    <footer>@microverse-inc</footer>
+    <div class="grid-container">
+      <header><h1>Leaderboard</h1></header>
```

Comment:
- [ ] Please spend a little bit more time to fix this overlapping between the scores and your footer, it'll just look better
![Screenshot from 2021-12-16 20-53-02](https://user-images.githubusercontent.com/62705947/146440205-c4ca15a3-a0f2-4fac-a100-61cc6f003a86.png)

Reply:
Overlap issues are fixed & a media query is also added.

Thank you for the review.